src/Chapter1.hs

```diff
@@ -542,8 +542,13 @@ value after "=" where the condition is true.
 Casual reminder about adding top-level type signatures for all functions :)
 -}
 
-mid x y z = error "mid: not implemented!"
-
+mid x y z
```

Comment:
You are encouraged to write top-level type signatures for all functions. This helps for understanding function expectations better :slightly_smiling_face: Otherwise, it's type will be inferred and it might not be what you expect.